### DISCUSSION TITLE
Custom Controller Path

### DISCUSSION THREAD

**Discussion by bobbyaxe74** [adonisjs/core#1394] [Help] [answered]
In adonis 4.1 i was able to declare a custom path for my controllers but in 5.1 seems that the original designated path "app/Controllers/Http/" still gets appended to my custom defined path how do i change this cause what i currently have after defining my current path is ".../build/app/Controllers/Http/App/Http/Controllers/GeenaController" as stated in the error message and this is how my route was defined `Route.get('/geena','App/Http/Controllers/GeenaController.test')`

**Comment by thetutlage** [comment, marked as answer]
Yes, there have been some changes to this. Now, you need to prepend the `/` to tell the IoC container that you are passing an absolute namespace.

```ts
Route.get('geena','/App/Http/Controllers/GeenaController.test')
```

**Reply by bobbyaxe74** [reply]
thanks a lot
